src/uwtools/tests/config/formats/test_sh.py

```diff
@@ -41,20 +40,17 @@ def test_parse_include():
     assert len(cfgobj) == 5
 
 
-def test_sh(salad_base, tmp_path):
+def test_sh(salad_base):
     """
     Test that sh config load and dump work with a basic sh file.
     """
     infile = fixture_path("simple.sh")
-    outfile = tmp_path / "outfile.sh"
     cfgobj = SHConfig(infile)
     expected: Dict[str, Any] = {
         **salad_base["salad"],
         "how_many": "12",
     }
     assert cfgobj == expected
-    cfgobj.dump(outfile)
-    assert filecmp.cmp(infile, outfile)
```

Comment:
An output file would no longer be identical to the input, because the input contains a comment and an executable statement that are ignored during parsing. See fixture file, below.